OMGVA PoS/Data layer/DTOs/Tax/CreateTaxRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OMGVA_PoS.Data_layer.DTOs
+{
+    public class CreateTaxRequest
+    {
+
+        [Required(ErrorMessage = "TaxType is required.")]
+        [MinLength(1, ErrorMessage = "TaxType must be at least 1 character.")]
+        public string TaxType { get; set; }
+        
+        [Required(ErrorMessage = "Percent is required.")]
```

Comment:
This automatically adds validation for the fields, i.e. 

Instead of manually checking and returning `BadRequest("Some bad request error");` 
![image](https://github.com/user-attachments/assets/7d23aa19-2b23-4008-8035-412353b3fdce)

We would automatically get a BadRequest without any code in controller
![image](https://github.com/user-attachments/assets/7c2369de-5b2e-459e-84da-9d6074f1290c)